tests/components/autocomplete_dropdown.test.ts

```diff
@@ -312,6 +337,30 @@ describe("Autocomplete parenthesis", () => {
     expect(model.getters.getComposerSelection()).toEqual({ start: 5, end: 5 });
   });
 
+  test.each(["Enter", "Tab"])(
+    "=SUM( + edit SUM with autocomplete does not add left parenthesis",
+    async (buttonkey) => {
+      await typeInComposerGrid("=SUM(");
+      // go behind the letter "M"
+      model.dispatch("STOP_COMPOSER_RANGE_SELECTION");
+      model.dispatch("CHANGE_COMPOSER_CURSOR_SELECTION", { start: 4, end: 4 });
+      await nextTick();
+      // show autocomplete
+      await typeInComposerGrid("I", false);
```

Comment:
actually, the 'sumif' function is not part of the functionregistry in this file,  see `beforeAll` at line 238, this means that you should **not** have any autocomplete suggestion once you add the `I` in the formula name. you should either use the compone t behaviour to do everything, instead of dispatching commands in the model directly, or maybe consider that the test is not at the right place in the file?

Reply:
L272, I have already added `SUMIF` in functionRegisty for `Autocomplete Parenthesis` set of test cases.